libparsec/src/invite.rs

```diff
@@ -384,20 +488,26 @@ pub async fn claimer_user_in_progress_1_do_signify_trust(
     let (cancel_requested, _canceller_guard) = listen_canceller(canceller)?;
     libparsec_platform_async::select2_biased!(
         res = work => res,
-        _ = cancel_requested => Err(ClaimInProgressError::Cancelled),
+        _ = cancel_requested => {
+            if canceller_ctx.cancel().await.is_err(){
```

Comment:
Add a commentto explain the error is dropped because this is best effort here

Reply:
e5dc2bed6a1b059894901854e3df8e15d765e1b0